tests/conftest.py

```diff
@@ -98,6 +110,73 @@ def ceph_pool_factory_class(request):
     return ceph_pool_factory_fixture(request)
 
 
+@pytest.fixture(scope="session", autouse=True)
+def log_ocs_version(cluster):
+    """
+    Fixture handling version reporting for OCS.
+
+    This fixture handles alignment of the version reporting, so that we:
+
+     * report version for each test run (no matter if just deployment, just
+       test or both deployment and tests are executed)
+     * prevent conflict of version reporting with deployment/teardown (eg. we
+       should not run the version logging before actual deployment, or after
+       a teardown)
+
+    Version is reported in:
+
+     * log entries of INFO log level during test setup phase
+     * ocs_version file in cluster path directory (for copy pasting into bug
+       reports)
+    """
+    cluster_version, image_dict = get_ocs_version()
+    file_name = os.path.join(
+        config.ENV_DATA['cluster_path'],
+        "ocs_version." + datetime.now().isoformat())
+    with open(file_name, "w") as file_obj:
+        report_ocs_version(cluster_version, image_dict, file_obj)
+    log.info("human readable ocs version info written into %s", file_name)
+
+
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
This fixture already exists in this file - https://github.com/red-hat-storage/ocs-ci/blob/9c9a814c7a847a4bee9b84112640c271dd2e9118/tests/conftest.py#L703

Reply:
Fixed